shell/lz-bridge/anvil-deployment/lib/common.sh

```
@@ -0,0 +1,200 @@
+  log "Starting Anvil fork of '$fork_alias' on $RPC (CUPS=$ANVIL_CUPS, backoff=${ANVIL_BACKOFF_MS}ms)"
+  anvil \
+    --fork-url "$fork_alias" \
+    --port "$PORT" \
+    --auto-impersonate \
+    --compute-units-per-second "$ANVIL_CUPS" \
+    --fork-retry-backoff "$ANVIL_BACKOFF_MS" \
+    --timeout 60000 \
+    > "$LOG_DIR/anvil-$fork_alias.log" 2>&1 &
+  ANVIL_PID=$!
+  local i
+  for i in $(seq 1 90); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)